### DISCUSSION TITLE
"For you" should be an automation and metrics tracking tool for personal along with enterprise task of the Github ecosystem

### DISCUSSION THREAD

**Discussion by horvatic** [community/community#13635] [Other Feature Feedback, Questions, & Ideas]
`For you` should be reworked as so a tool an engineer can use keep track of: recent deployment, status of those deployments, release made for a repo, etc. Anything about the person, or enterprise level that would help better keep track of all the life cycles going on in the Github environment. The engineer should also be able to choose what they want to track, and get repos on.

Last there should be a HUGE focus on automation. I should be able to setup a bot that will be able to gather all these metrics very easy without a PAT hack. 

This tool will be extremely useful on the DevOps level keep tracking of everything on Github the enterprise / personal dev is used.

**Comment by jfullstackdev** [comment]
> The engineer should also be able to choose what they want to track, and get repos on.

I think for now this is done,

you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
